Community/Tdarr_Plugin_TN10_HEVCAAC.js

```diff
@@ -0,0 +1,727 @@
+// jshint esversion: 6
```

Comment:
Could you explain the differences between your plugin and the current version of `Tdarr_Plugin_JB69_JBHEVCQSV_MinimalFile`? All I can really see is a bunch of renamed variables and a slightly different way to calculate a video bitrate.

Reply:
It's a pretty significant simplification to the user facing options, in my opinion it's a much more intuitive way of setting up transcode parameters, and the user facing parameters are only ones that they should be changing. I also fixed an issue the original script had with transcoding older codecs such as `divx` and `xvid` and eliminated `target_framerate` as that did not work. 

![image](https://user-images.githubusercontent.com/11021632/169524484-32cec37b-8b94-40dd-88f2-7d6a0c7c0091.png)

vs 

![image](https://user-images.githubusercontent.com/11021632/169525344-c8c29659-1360-499c-80fa-489ad82e8739.png)

Reply:
I was under the impression that the core of the code and inputs could not be changed once published as it could/would be a breaking change for exiting users, and that only bugs could be fixed.